### DISCUSSION TITLE
Help me my push error

### DISCUSSION THREAD

**Discussion by grucare** [community/community#65415] [New to GitHub] [answered]
I'm not familiar to Github. any one teach me how to push.

This is my issues
I'm going to push to new repository from existing one.
So I've used this steps :
git remote add origin https://github.com/lobcare/lobcare.git
git branch -M main
git push -u origin main

But doesn't work

**Comment by queenofcorgis** [comment, marked as answer]
Hi there, 

Have you tried removing origin from your first step and just leaving the URL?

**Comment by deemaaldahrouj** [comment]
when i push my code it give me the error remote: Invalid username or password. and i can't solve the problem
